audioread/gstdec.py

```diff
@@ -361,6 +361,8 @@ def next(self):
             raise StopIteration
         return val
 
+    __next__ = next
```

Comment:
Looks good. Can we please get a comment explaining that this is for Python 3/2 compatibility?